### PR TITLE
Fixed Category Creation bug

### PR DESCRIPTION
**Pull request by noglows**
Fixed the category creation bug and added Rspec and Cucumber tests for the bug

### CODE REVIEW

File: features/create_category.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  In order to share my thoughts with the world
+  I want to be able to add categories to my blog posts
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Successfully go to admin category creation page
+    Given I am on the new admin category page
+    When I fill in "category_name" with "Test Category"
```
Comment (kariabancroft) [inline features/create_category.feature]: What about the other fields?

File: spec/controllers/admin/categories_controller_spec.rb [hunk 2]
```diff
@@ -34,6 +34,18 @@
     end
   end
 
+  describe "test_new" do
+    before(:each) do
+      get :new
+    end
+
+    it 'should render template new' do
+      assert_template 'new'
```
Comment (kariabancroft) [inline spec/controllers/admin/categories_controller_spec.rb]: Should there be a specific status returned?

**Comment by noglows** [comment]
@kariabancroft I updated the tests based on your comments. Added test for the other fields to Cucumber test and added a response status check to the Rspec tests.

**Comment by kariabancroft** [comment]
Looks good to close!

**Pull request closed without merging**